test-utils/loadtester/README.md

```diff
@@ -3,61 +3,52 @@
 This tool can be used to test a local or remote set of nodes. It submits transactions at a given rate and monitors
 the output TPS by periodically requesting the most recent block from the leading nodes.
 
-## Example
+## Example of a local testnet
 
-The following is an example of how to crash a locally running TestNet by saturating it with transactions.
-As of 2019-04-16, this has not been fixed.
+The following is an example of attempt to crash a locally running TestNet by saturating it with transactions.
+As of 2019-09-17, this has been fixed.
```

Comment:
I don't think we need this line since it is now fixed.

Reply:
Sounds good, will do